### ISSUE TITLE
Preview of serviceset does not work

### ISSUE THREAD

**Issue by tobiasvdk** [Icinga/icingaweb2-module-director#1069]
The preview of a serviceset does not show the rendered config.

## Expected Behavior
Show the complete rendered config of all services in the service set.

## Current Behavior
No config is displayed

## Steps to Reproduce (for bugs)

1. Create serviceset
2. Create a service in the serviceset
3. Open the "Preview" tab
4. No config is shown

## Context

I want to verify that the generated config is correct.

## Your Environment

* Director version (System - About): 1d683f972d9ec8651f94eccee121d5e74a5b0e32
* Icinga Web 2 version and modules (System - About): 2.4.1
* Icinga 2 version (`icinga2 --version`): 2.6.3
* Operating System and version: Debian Jessie
* Webserver, PHP versions: 2.4.10, 5.6.30

**Comment by Thomas-Gelf** [comment]
HI @tobiasvdk,

does your set have an assign rule? As long as no one is using it, the Set isn't rendered at all - as it does "not exist".

**Comment by lazyfrosch** [comment]
I guess we should improve that and output at least a comment about an unused set...

**Comment by Thomas-Gelf** [comment]
I'll give it a look, as I'm working on Service Sets today.

**Comment by tobiasvdk** [comment]
Yes, you're right - it works. I grabbed the one and only service set which (still) doesn't applied.

**Comment by lazyfrosch** [comment]
We still should make sure that a user understands whats happening.

Examples:

* rendering the set in disabled mode, unless it has an assign rule, or direct host reference
* rendering some empty comment when the service set doesn't have services yet
